pykickstart/commands/updates.py

```diff
@@ -71,3 +73,46 @@ def parse(self, args):
             self.url = _ns.updates[0]
 
         return self
+
+
+class F34_Updates(F7_Updates):
+    removedKeywords = F7_Updates.removedKeywords
+    removedAttrs = F7_Updates.removedAttrs
+
+    def __str__(self):
+        retval = KickstartCommand.__str__(self)
+
+        if self.url:
+            retval += "updates %s\n" % self.url
+
+        return retval
+
+    def _getParser(self):
+        op = KSOptionParser(
```

Comment:
Pykickstart doesn't support redefined positional arguments. I've tried several workarounds, but it was easier to redefine the option parser.